bucket/audmes.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "20211104",
+    "description": "AUDio MEasurement System - PC based Oscilloscope and Spectrum analyzer using sound card",
+    "homepage": "https://sourceforge.net/projects/audmes/",
+    "license": "GPLv2",
```

```suggestion
    "license": "GPL-2.0-or-later",
```
We use [SPDX license identifiers](https://spdx.org/licenses/) as our standard.